lib/Login.dart

```diff
@@ -103,7 +103,7 @@ class _LoginState extends State<Login> {
       name = googleSignIn.currentUser.displayName;
       email = googleSignIn.currentUser.email;
       imageUrl = googleSignIn.currentUser.photoUrl;
-      return Profile();
+      return Current();
```

Comment:
Make sure you are update with master. This should link to the Homepage. If you want to make the Current Ride page accessible for my viewing, integrate it into the app by having it appear when you actually click the current ride, for example. Integrating it fully into the flow of the app is always better than replacing the homepage because it does not affect master for anything else. If you have questions about how to get to this page or about app flow in general, contact the designer who worked on the design.